kyuubi-metrics/src/main/scala/org/apache/kyuubi/metrics/PrometheusReporterService.scala

```diff
@@ -35,12 +36,21 @@ class PrometheusReporterService(registry: MetricRegistry)
 
   // VisibleForTesting
   private[metrics] var httpServer: Server = _
+  private[metrics] var connector: ServerConnector = _
```

Comment:
```suggestion
  private[metrics] var httpServerConnector: ServerConnector = _
```
To follow the naming conversion with existed `httpServer` variable.